modin/core/dataframe/pandas/dataframe/utils.py

```diff
@@ -479,6 +505,178 @@ def _index_to_df_zero_copy(
         return index_data
 
 
+@_inherit_docstrings(ShuffleSortFunctions)
+class ShuffleResample(ShuffleSortFunctions):
+    def __init__(
+        self,
+        modin_frame: "PandasDataframe",
+        columns: Union[str, list],
+        ascending: Union[list, bool],
+        ideal_num_new_partitions: int,
+        resample_kwargs: dict,
+        **kwargs: dict,
+    ):
+        resample_kwargs = resample_kwargs.copy()
+        rule = resample_kwargs.pop("rule")
+
+        if resample_kwargs["closed"] is None:
+            # this rule regarding the default value of 'closed' is inherited
```

Comment:
```suggestion
            # this rule regarding the default value of 'close to right' is inherited
```
?

Reply:
here I refer to the default value of `resample(closed=...)`, so I would preserve the original naming